docker/devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# The devcontainer Dockerfile to set up an hermetic enviornment with all needed dependencies.
+
+FROM rayproject/ray-deps:2023-06-26-cu116-aarch64
```

Comment:
how would you update this? this is not building from source.

for example HEAD's default gpu image is already cu118 now.

also, when this is on mac apple silicon, I do not know why cuda is being used. as I don't think cuda driver would work.